src/show.cpp

```diff
@@ -118,6 +121,13 @@ void Show::printProperty(const char* title, const char* name,
     }
 }
 
+void Show::printProperty(const char* title, const char* name,
+                         const Dbus::Properties& properties) const
+{
+    printProperty(title, name, properties,
+                  std::make_pair("Enabled", "Disabled"));
```

Comment:
It seems to me they have wrong order either here or at the string with my previous comment.